docs/sources/reference/components/beyla/beyla.ebpf.md

```diff
@@ -156,6 +165,39 @@ Name               | Type           | Description
 `owner_name`       | `string`       | Regular expression of Kubernetes owners of running Pods to match.                                           | `""`    | no
 `pod_labels`       | `map(string)`  | Key-value pairs of labels with keys matching Kubernetes Pods with the provided value as regular expression. |  `{}`   | no
 
+### prometheus block
+
+This block configures which metrics Beyla collects.
+
+Name              | Type           | Description                                                    | Default           | Required
+------------------|----------------|----------------------------------------------------------------|-------------------|---------
+`features`        | `list(string)` | List of features to enable for the metrics.         | `["application"]` | no
+`instrumentations`| `list(string)` | List of instrumentations to enable for the metrics. | `["*"]`           | no
+`network`         | `map(string)`  | Configures network metrics options for Beyla.                  | `{}`              | no
+
+`features` is a list of features to enable for the metrics. The following features are available:
+
+- `application` exports application-level metrics.
+- `application_span`exports application-level metrics in traces span metrics format.
+- `application_service_graph` exports application-level service graph metrics.
+- `application_process` exports metrics about the processes that run the instrumented application.
+- `network` exports network-level metrics.
+
+`instrumentations` is a list of instrumentations to enable for the metrics. The following instrumentations are available:
+
+- `*` enables all `instrumentations`. If `*` is present in the list, the other values are ignored.
+- `http` enables the collection of HTTP/HTTPS/HTTP2 application metrics.
+- `grpc` enables the collection of gRPC application metrics.
+- `sql` enables the collection of SQL database client call metrics.
+- `redis` enables the collection of Redis client/server database metrics.
+- `kafka` enables the collection of Kafka client/server message queue metrics.
+
+`network` configures network metrics options for Beyla. The following options are accepted:
```

Comment:
is `network` intended to be a block or an attribute?
- if in the future there will be more options to configure the network metrics, then it makes sense that this is a block but it needs to be documented as such (it should be in the block table `metrics > network` and have its own part just like the `attributes > kubernetes` for example)
- if it's only something that can be toggled and it is intended that it stays that way then it should just be a boolean attribute and should be renamed something like `network_enabled` or network_metrics_enabled`

Reply:
It's a block. In the future there should be more options to configure the network metrics: https://grafana.com/docs/beyla/latest/network/config/

Reply:
ok then once you adapt the doc for it we will be good to go from my POV. Also make sure you remove the `network` entry from the metrics table above (this is for attributes).

Reply:
ok! done